Source/Core/VideoBackends/OGL/OGLConfig.cpp

```diff
@@ -531,8 +538,22 @@ bool PopulateConfig(GLContext* m_main_gl_context)
   if (g_ogl_config.max_samples < 1 || !g_ogl_config.bSupportsMSAA)
     g_ogl_config.max_samples = 1;
 
-  g_ogl_config.bSupportsShaderThreadShuffleNV =
-      GLExtensions::Supports("GL_NV_shader_thread_shuffle");
+  g_ogl_config.bSupportsKHRShaderSubgroup = GLExtensions::Supports("GL_KHR_shader_subgroup") &&
+                                            ((g_ogl_config.eSupportedGLSLVersion >= Glsl450 &&
+                                              g_ogl_config.eSupportedGLSLVersion < GlslEs300) ||
+                                             g_ogl_config.eSupportedGLSLVersion >= GlslEs320);
```

Comment:
Maybe this would be more clear (I checked with lint for the formatting)
```suggestion
  g_ogl_config.bSupportsKHRShaderSubgroup =
      GLExtensions::Supports("GL_KHR_shader_subgroup") &&
      (g_ogl_config.bIsES ? g_ogl_config.eSupportedGLSLVersion >= GlslEs320 :
                            g_ogl_config.eSupportedGLSLVersion >= Glsl450);
```